packages/ui-components/src/components/advanced-date-select-dropdown/advanced-date-select-dropdown.tsx

```diff
@@ -38,7 +38,7 @@ export class KvAdvancedDateSelectDropdown implements IAdvancedDateSelectDropdown
 	/** @inheritdoc */
 	@Prop({ reflect: false }) relativeTimeConfig?: ICalendarAdvanceRelativeTimeConfig;
 	/** @inheritdoc */
-	@Prop({ reflect: false }) selectedTimezone?: string;
+	@Prop({ reflect: false }) selectedTimezone?: string = getDefaultTimezone() ?? 'UTC';
```

Comment:
Move this `UTC` to config as `FALLBACK_TIMEZONE` or directly to the `getDefaultTimezone()`.